test/select_tests.cpp

```diff
@@ -805,3 +805,123 @@ TEST(IntervalLiteral) {
     }
   }
 }
+TEST(LockingClauseWithoutWaitPolicy) {
```

Comment:
Please add an empty line between the closing brace and the new test case.

Reply:
Please also check
- the number of elements in `lockings`
- `tables` (of the struct `LockingClause`)